adidas-ascii-logo/swift/Tests/adidas-ascii-logoTests/XCTestManifests.swift

```diff
@@ -0,0 +1,9 @@
+import XCTest
+
+#if !os(macOS)
+public func allTests() -> [XCTestCaseEntry] {
+    return [
+        testCase(adidas_ascii_logoTests.allTests),
+    ]
+}
+#endif
```

Comment:
It is recommended to add a blank line at the end of the file.

Reply:
Done